extensions/amp-story/validator-amp-story.protoascii

```diff
@@ -419,6 +420,29 @@ tags: {  # amp-story-bookend (json)
   }
   attr_lists: "nonce-attr"
 }
+tags: {  # <amp-story-social-share>
+  html_format: AMP
+  tag_name: "AMP-STORY-SOCIAL_SHARE"
```

Comment:
AMP-STORY-SOCIAL-SHARE

Reply:
Thanks! That was a very hidden typo, will write a validation test 😁